shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -209,7 +209,16 @@
           .required(false)
           .hasArg(false)
           .desc("Use passive-cache or direct cache request,"
-              + " turn on if you want to use the old passive cache implementation")
+              + " turn on if you want to use the old passive cache implementation. "
+              + "Passive-cache is default when there's no option set or both options are set.")
+          .build();
+  private static final Option DIRECT_CACHE_OPTION =
```

Comment:
a little strange to have two options, but I guess we are doing this for backcompat.  Does java support something like --passive-cache=false?

Reply:
no you still have to specify the value here. If we don't care about the backcompat we can change to use boolean

Reply:
emmm... it's a little confused and complicated to have two options.
I will prefer to have `--passive-cache = true` or `default` do passive-cache.
have `--passive-cache=false` do direct-cache.

Reply:
> @yuzhu @LuQQiu let's assume there are not many users have this compatibility issue(have passive-cache flag turned on) and use boolean value for later usage. I would update this PR if there's no objection to this
I think @LuQQiu 's suggestion works well here. It provides good default behavior, and frankly no one should be using direct cache until we fix it.